ci/prow/make_config.go

```diff
@@ -975,6 +999,17 @@ func executeTemplate(name, templ string, data interface{}) {
 	}
 }
 
+// Multi-value flag parser.
+
+func (a *stringArrayFlag) String() string {
+	return strings.Join(*a, ", ")
+}
+
+func (a *stringArrayFlag) Set(value string) error {
```

Comment:
this function always return nil, is it necessary?

Reply:
It doesn't fail because there's no sanity check of the values. I can't break the contract otherwise flag parsing will break.